SCons/Environment.xml

```diff
@@ -1042,9 +1026,8 @@ A now-deprecated synonym for
 Specifies that all up-to-date decisions for
 targets built through this construction environment
 will be handled by the specified
-<varname>function</varname>.
-The
-<varname>function</varname>
+<parameter>function</parameter>.
+<parameter>function</parameter>
```

Comment:
Shouldn't the second one here be "The <..."

Reply:
it's saying the `function` argument can also be a string. "The" isn't right, but I'll adjust wording.